components/bl-stopwatch/component.json

```
@@ -35,13 +35,31 @@
       "settingTooltip": "Allows to determine time format of stopwatch"
     },
     {
-      "type": "number",
-      "name": "tickRate",
-      "label": "Tick Rate",
+      "type": "select",
+      "name": "decimalPlaces",
+      "label": "Decimal Places",
       "showInSettings": true,
       "defaultValue": "3",
+      "options": [
+        {
+          "value": "0",
+          "label": "0"
+        },
+        {
+          "value": "1",
+          "label": "1"
+        },
+        {
+          "value": "2",
+          "label": "2"
+        },
+        {
+          "value": "3",
+          "label": "3"
+        }
+      ],
```

```suggestion
      "type": "select",
      "name": "tickRate",
      "label": "Decimal Places",
      "showInSettings": true,
      "defaultValue": "1",
      "options": [
        {
          "value": "0",
          "label": "1000"
        },
        {
          "value": "1",
          "label": "100"
        },
        {
          "value": "2",
          "label": "10"
        },
        {
          "value": "3",
          "label": "1"
        }
      ],
```

Leave readable label as "Decimal Places" and use `tickRate` in code without any calculations.

---

I don't understand why we need to change the label if then the user will see "Decimal Places: 1000".

---

Make mistake. Labels will be the same. Changed only values.

---

But for determining the number of decimal places it will not be very suitable for us. I suggest leaving value as a label, and creating a map for interval:
`
{
  '0': 1000,
  '1': 100,
  '2': 10,
  '3': 1,
};
`

---

> it will not be very suitable for us.

Is there any arguments? 

I don't see why it could be more convenient, but if you want to leave it:

```
const TickRateInMs = {
  '0': 1000,
  '1': 100,
  '2': 10,
  '3': 1,
};
```